sendKindle.py

```diff
@@ -210,7 +217,6 @@ def main():
     '''Run the main program'''
     try:
         kindle = SendKindle()
-        kindle.send_mail()
```

Comment:
Why is this removed?

Reply:
By mistake, sorry, I was editing directly the code from pip, and there it seems there's an older version where this function was missing.

Reply:
Hmm, there should be the same version:
https://pypi.org/project/sendKindle/#history
No problem.